vm-aws-python/__main__.py

```diff
@@ -0,0 +1,87 @@
+import pulumi
+import pulumi_aws as aws
+
+# Get some configuration values or set default values.
+config = pulumi.Config()
+instance_type = config.get("instanceType")
+if instance_type is None:
+    instance_type = "t3.micro"
+vpc_network_cidr = config.get("vpcNetworkCidr")
+if vpc_network_cidr is None:
+    vpc_network_cidr = "10.0.0.0/16"
+
+# Look up the latest Amazon Linux 2 AMI.
+ami = aws.ec2.get_ami(filters=[aws.ec2.GetAmiFilterArgs(
+        name="name",
+        values=["amzn-ami-hvm-*"],
+    )],
+    owners=["137112412989"],
```

Comment:
I have found that using `"amazon"` is much more intuitive versus `"137112412989"`. (They resolve to the same.)

Reply:
Done